plugins/org.eclipse.glsp.api/src/org/eclipse/glsp/api/action/kind/InitCreateOperationAction.java

```diff
@@ -0,0 +1,55 @@
+/********************************************************************************
+ * Copyright (c) 2020 EclipseSource and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * https://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+package org.eclipse.glsp.api.action.kind;
+
+import java.util.Map;
+
+import org.eclipse.glsp.api.action.Action;
+
+public class InitCreateOperationAction extends Action {
```

Comment:
What is an `InitCreateOperationAction`?

Reply:
This action contains the information that the client/tools needs to initialize a CreateOperationAction for a certain element type. (i.e element type id and operation kind).

Reply:
This is basically the information that was previously stored in the `Operation` interface

Reply:
Right, makes sense... I'm also wondering if we come up with a better name here. Maybe a task for our architecture committee meeting :)